packages/polkadot/src/polkadot.childBounties.e2e.test.ts

```diff
@@ -0,0 +1,11 @@
+import { polkadot } from '@e2e-test/networks/chains'
+import { baseChildBountiesE2ETests, registerTestTree, type TestConfig } from '@e2e-test/shared'
+
+const testConfig: TestConfig = {
```

Comment:
`export type TestConfig = RelayTestConfig | ParaTestConfig`

You need to explicitly specify `ParaTestConfig`.